site/data/docs/custom-avatar.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Custom Avatar
+description: Customizing your app's user avatar
+---
+
+# Custom Avatar
+
+## Customizing your app’s user avatar
+
+By default we provide an avatar for users in case their ENS image is not set, but you can customize this by providing your own avatar component. You can import the `AvatarComponent` type if using TypeScript:
+
+```tsx
+import {
+  RainbowKitProvider,
+  AvatarComponent,
+} from '@rainbow-me/rainbowkit';
+import { generateColorFromAddress } from './utils';
+
+const CustomAvatar: AvatarComponent = ({ address, ensImage, size }) => {
+  const color = generateColorFromAddress(address);
+  return ensImage ? (
+    <img src={ensImage} width={size} height={size} />
```

Comment:
I think it'd be a good idea to add `style={{ borderRadius: 999 }}` to both of these so that devs get circles when they copy+paste this.

Reply:
Added, this is now in `custom-avatars.mdx`